pvfactors/pvarray.py

```diff
@@ -133,10 +133,13 @@ class Array(ArrayBase):
         center [meters]
     :param float pvrow_width: width of PV rows, in the considered 2D
         dimension [meters]
-    :param float array_tilt: tilt angle of the whole array. All PV rows must
-        have the same tilt angle [degrees]
-    :param float array_azimuth: azimuth angle of the whole array. All PV
-        rows must have the same azimuth angle [degrees]
+    :param float surface_tilt: Surface tilt angles in decimal degrees.
+        surface_tilt must be >=0 and <=180.
+        The tilt angle is defined as degrees from horizontal
+        (e.g. surface facing up = 0, surface facing horizon = 90)
+    :param float axis_azimuth: A value denoting the compass direction along
+        which the axis of rotation lies. Measured in decimal degrees East of
+        North.
```

Comment:
Why define a new parameter? Can't we just use surface_azimuth here?

Reply:
Sorry I forgot to fix this docstring, thanks for catching this!